dataflow/manual/content.tex

```diff
@@ -1344,7 +1338,8 @@ \subsubsection{Desugaring}
 
     \item Once we decided to make conversion nodes explicit it made
       sense to desugar compound assignments.  A compound assignment
-      like \begin{verbatim}Integer i; i += 3;\end{verbatim} performs
+      like \begin{verbatim}Integer i; i += 3;\end{verbatim} or
+      \begin{verbatim}String s; s += "a";\end{verbatim} performs
```

Comment:
This sentence talks about boxing of `i`, which doesn't make sense in the string example.
Maybe string compound concatenation should be a separate item in this list?

Reply:
Yeah, will start a new item here, I think it is a suitable place.